frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -52,23 +56,16 @@ body {
 }
 
 .ThemeNight.EmbedFrame .fullscreenNightText {
-  color: var(--mb-color-text-white-alpha-85);
   transition: color 1s linear;
 }
 
 .ThemeNight.EmbedFrame svg text {
-  fill: var(--mb-color-text-white-alpha-85) !important;
   stroke: none !important;
 }
 
 .ThemeNight.EmbedFrame .DashCard .Card {
+  border: 1px solid var(--mb-color-border);
```

Comment:
I just saw this, this was a mistake, it should stay below `background-color`, but I don't want to trigger more CI checks unless there are more places to fix 🤦

Reply:
Fixed in https://github.com/metabase/metabase/pull/45134/commits/65f8e1aaf622c4072d3dd48405d36060ebabd241